### PR TITLE
Add(route): add 豆瓣招聘 and change to V2

### PR DESCRIPTION
**Pull request by Fatpandac**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/douban/jobs/social
/douban/jobs/campus
/douban/jobs/intern
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [X] New Route
- [X] Documentation
  - [X] CN
  - [ ] EN
- [ ] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] 日期和时间 date and time
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by AgFlore** [comment]
miantainer 拼错了（。

**Comment by Fatpandac** [comment]
> miantainer 拼错了（。

感谢指正，新年快乐呀！

File: lib/v2/douban/maintainer.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    '/book/rank/:type?': ['xyqfer', 'queensferryme'],
+    '/doulist/:id': ['LogicJake'],
+    '/people/:userid/status/:routeParams': ['alfredcai'],
+    '/replies/:uid': ['nczitzk'],
+    '/replied/:uid': ['nczitzk'],
+    '/topic/:id/:sort?': ['LogicJake'],
+    '/channel/:id/:nav?': ['umm233'],
+    '/channel/:id/subject/:nav': ['umm233'],
+    '/people/:userid/wish/:routeParams?': ['exherb'],
+    '/jobs/:type': ['Fatpandac'],
```
Comment (TonyRL) [inline lib/v2/douban/maintainer.js:26]: Please sort the routes in alphabetical order

File: lib/v2/douban/router.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    router.get('/explore/column/:id', require('./other/explore_column'));
+    router.get('/people/:userid/status/:routeParams?', require('./people/status.js'));
+    router.get('/people/:userid/wish/:routeParams?', require('./people/wish.js'));
+    router.get('/replies/:uid', require('./other/replies'));
+    router.get('/replied/:uid', require('./other/replied'));
+    router.get('/topic/:id/:sort?', require('./other/topic.js'));
+    router.get('/channel/:id/:nav?', require('./channel/topic.js'));
+    router.get('/channel/:id/subject/:nav', require('./channel/subject.js'));
+    router.get('/celebrity/:id/:sort?', require('./other/celebrity.js'));
+    router.get('/jobs/:type', require('./other/jobs.js'));
```
Comment (TonyRL) [inline lib/v2/douban/router.js:26]: Please sort the routes in alphabetical order

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/douban/router.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+module.exports = function (router) {
+    router.get('/book/latest', require('./other/latest_book'));
+    router.get('/book/rank/:type?', require('./book/rank'));
+    router.get('/bookstore', require('./other/bookstore'));
+    router.get('/celebrity/:id/:sort?', require('./other/celebrity.js'));
+    router.get('/channel/:id/:nav?', require('./channel/topic.js'));
+    router.get('/channel/:id/subject/:nav', require('./channel/subject.js'));
```
Comment (TonyRL) [inline lib/v2/douban/router.js:7]:
```suggestion
    router.get('/channel/:id/subject/:nav', require('./channel/subject.js'));
    router.get('/channel/:id/:nav?', require('./channel/topic.js'));
```
This two need to be swapped

**Review by TonyRL — CHANGES_REQUESTED** [review]

**Pull request merged**